src/main/java/com/exactpro/th2/common/schema/grpc/router/impl/DefaultGrpcRouter.java

```diff
@@ -110,14 +128,17 @@ protected void validateArgs(Object[] args) {
     }
 
     protected AbstractStub<?> getGrpcStubToSend(Class<?> proxyService, Message message) throws ClassNotFoundException {
-        var grpcStubAnn = proxyService.getAnnotation(GrpcStub.class);
+        var grpcStubAnnOld = proxyService.getAnnotation(GrpcStub.class);
+        var grpcStubAnnNew = proxyService.getAnnotation(com.exactpro.th2.service.generator.service.annotation.GrpcStub.class);
```

Comment:
If we are migrating to the new annotation we probably should import it instead of the old one.